velox/functions/prestosql/fuzzer/AggregationFuzzerTest.cpp

```diff
@@ -55,6 +55,12 @@ DEFINE_string(
     "source of truth. Otherwise, use DuckDB. Example: "
     "--presto_url=http://127.0.0.1:8080");
 
+DEFINE_uint32(
+    req_timeout,
+    1000,
+    "Timeout in milliseconds of http request made to reference DB, "
+    " such as Presto. Example: --req_timeout=2000");
```

Comment:
“Timeout in milliseconds for HTTP requests made to reference DB,  such as Presto. Example: --req_timeout=2000” ？